routes/api/util/form-validation/edit.js

```diff
@@ -0,0 +1,27 @@
+const Validator = require('validator');
+const isEmpty = require('is-empty');
+
+/**
+ * Takes in edit form input and checks for validity
+ * @param requestBody the data object holding all form inputs as fields
+ */
+function validateEditInput(requestBody) {
```

Comment:
One thing I just realized is if they submit the form and all inputs are empty, then nothing would update and we'd write to MongoDB for no reason. Maybe loop through the `requestBody` before checking any of the fields and iterate an `emtpyCount` variable, and if the emptyCount is the same as the requestBody size then we can add an error that says everything empty